Lib/test/test_float.py

```diff
@@ -768,6 +768,8 @@ def test_format(self):
         self.assertEqual(format(x, '<21._f'), '123456.123_456       ')
         self.assertEqual(format(x, '+.11_e'), '+1.234_561_234_56e+05')
         self.assertEqual(format(x, '+.11,e'), '+1.234,561,234,56e+05')
+        self.assertEqual(format(x, '019_._f'), '000_123_456.123_456')
+        self.assertEqual(format(x, '021_.10_f'), '123_456.123_456_000_0')
```

Comment:
Please add also tests for similar but different formats:
* without fill or with different fill character
* with different alignments
* with "e" instead of "f".

There are already tests for `'>21._f'` and `'>21._f'`, so we need to test `'>021._f'`, `'<021._f'`, `'021._f'`, `'21._f'`, `' 21._f'`, `'021._e'`. And something with precision.